libs/utils/src/auth.rs

```diff
@@ -123,9 +143,9 @@ impl JwtAuth {
             }
         }
         if let Some(res) = res {
-            res.map_err(anyhow::Error::new)
+            res.map_err(|e| AuthError(Cow::Owned(e.to_string())))
```

Comment:
I was going to ask about this but later on it's formatted static `ApiError::Forbidden("JWT ...".to_string())` -- is this form ever logged by something? I don't at least see it in case it was meant to be logged.

Reply:
This is now logged by query error logging.

Reply:
For ApiError [8948a7f](https://github.com/neondatabase/neon/pull/5823/commits/8948a7f0ac03a253317bed87a93cf79070f6acf5) logs it now in addition to the ApiError conversion time logging.